lib/transform_utils.js

```diff
@@ -2,7 +2,8 @@
 
 var _ = require('lodash');
 
-var top_level_types = ["story", "video", "image", "gallery", "revision"];
+var top_level_types = ["story", "video", "image", "gallery", "results", "audio", "redirect"];
```

Comment:
Should we have lost `revision` here?  Tests pass, so maybe it was in erroneously before, but I didn't see mention of it elsewhere here or in the proposal.